packages/api-client/src/http/BackendErrorMapper.ts

```diff
@@ -236,16 +219,20 @@ export function mapBackendError(error: BackendError): BackendError {
   // 1) Message-specific variant
   const messageVariantHandler = messageVariantHandlers[code]?.[label]?.[message];
   if (messageVariantHandler) {
-    return messageVariantHandler({...error, label} as BackendErrorWithLabel);
+    const mapped = messageVariantHandler({...error, label} as BackendErrorWithLabel);
```

Comment:
we dont use "as" in out code please

I repeat myself here, but refer to a typeguard